test/strings.ts

```diff
@@ -25,3 +25,154 @@ test("test string join ordering", (assert) => {
   `);
   assert.end();
 });
+
+test("check length of hello", (assert) => {
```

Comment:
These may need touched depending on whether they go in before the async meat-grinder work Chris has going on.